### ISSUE TITLE
Various UI missing components

### ISSUE THREAD

**Issue by Cozzolino92** [LoneGazebo/Community-Patch-DLL#4422]
_1. Mod version: 4/30

_2. Mod list: CP + CBO only; InGame Editor

_3. Error description: Various UI missing components.
- "Swap Great Work" window shows only 1 city, trying to sort them will show other cities but always one at once
- Missing Unhappiness breakdown
- Unit supply cap missing
- Probably more but I can't tell as they are just not there and it's my first time with EUI

_4. Steps to reproduce: 
Using automatic installer, I used the Vox Populi with EUI setup option, then just started a game.
Deleting EUI manually did not fix the problem.

Notes: I wasn't even sure I should be posting this here. I've searched for the issue both here and on forums but this always comes up when there's a conflict between mods, yet I'm only using IGE and CP + CBO only. Am I doing something wrong? Can provide screenshots if needed.

**Comment by LoneGazebo** [comment]
Mod conflict. You can’t use CBO only with EUi.

**Comment by Cozzolino92** [comment]
Thanks for the rapid response, it makes sense now.
I deleted EUI from assets/mods folder, run the VP setup again and selected the basic option with CP + CBO. I proceeded to delete the cache folder and started another game. While all the info in the panels are now readily available, the icon on my city is all messed up. Should I just clean install or is there a quick fix?
![civ5screen0006](https://user-images.githubusercontent.com/39533968/40385550-42346ae8-5e07-11e8-9408-5cfcca768707.jpg)
